less/index.less

```diff
@@ -56,4 +53,168 @@ table {
 html, body {
     height: 100%;
 }
+html{
+	font-size:62.5%;
+}
+p{
+	font-size:@baseFontSize;
+}
+li{
+	font-size:@baseFontSize;
+}
+// Mixin declarations
+.center(){
+	display:flex;
+	justify-content: center;
+	align-items:center;
+}
+.aTagStyle{
+	font-size:@baseFontSize;
+	color:black;
+	text-decoration:none;
+}
+// these were mostly just for practice
+@baseFontSize:1.6rem;
+@h2FontSize:2.5rem;
+@bgColor:lightblue;
+
+// Home Page CSS
+// gotta love these single line comments!
+.container{
+	max-width:880px;
+	margin:0 auto;
+	@media(max-width:500px){
+		max-width:500px;
+	}
+}
+header{
+	width:100%;
+	nav{
+		.center;
+		justify-content:flex-end;
+		@media(max-width:500px){
+			.center;
+			flex-direction: column;
+		}
+		a{
+			.aTagStyle;
+			display:flex;
+			margin-right:5%;
+			&:hover{
+				text-decoration:underline;
+			}
+			@media(max-width:500px){
+				font-size:4rem;
+				padding-top:5%;
+				width:100%;
+				.center;
+				border-bottom:1px solid black;
+			}
+
+		}
+	}
+}
+.intro-container{
+	margin-top:5%;
+	padding:5%;
+	background-color:@bgColor;
+	.main-intro{
+		border-bottom:2px solid grey;
+		padding-bottom:2.5%;
+		img{
+			width:auto;
+			height:300px;
+			@media(max-width:500px){
+				width:100%;
+				height:auto;
+			}
+		}
+		h1{
+			font-size:5rem;
+		}
+		p{
+			line-height:4rem;
+			font-size:@baseFontSize + 0.5;
+		}
+	}
+	.lorem-text{
+		margin-top:3%;
+		margin-bottom:2%;
+	}
+}
+.team-up-btn{
+	.center;
+	width:150px;
+	height:40px;
+	border-radius:2px;
+	background-color:grey;
+	a{
+		.aTagStyle;
+		.center;
+	}
+}
+.qualifications-section{
+	width:100%;
+	display:flex;
+	justify-content: space-between;
+	padding-top:5%;
+	background-color:lightgrey;
+	@media(max-width:500px){
+		flex-direction: column;
+	}
+	.listen-up{
+		width:45%;
+		h2{
+			font-size:@h2FontSize;
+			margin-bottom:3%;
+		}
+		@media(max-width:500px){
+			width:100%;
+		}
+	}
+	.qualifications{
+		width:45%;
+		h2{
+			font-size:@h2FontSize;
+			margin-bottom:3%;
+		}
+		ul{
+			line-height:2rem;
+		}
+		@media(max-width:500px){
+			width:100%;
+		}
+	}
+}
+.work-history{
+	width:100%;
+	flex-direction:column;
+	background-color:@bgColor;
+	.center;
+	padding-bottom:5%;
+	h2{
+		font-size:@h2FontSize;
```

Comment:
I think if you have something like h2FontSize, which sounds very general, just have a h2 selector at the top level of your container and set it there